### PR TITLE
JJ hcmi branch

### PR DESCRIPTION
**Pull request by jjacobson95**
Updates:

HCMI Data updated. Sample.tsv updated with improve IDs and cancer type data.

[figshare_pull.py](https://github.com/PNNL-CompBio/candleDataProcessing/compare/JJ_hcmi_branch?expand=1#diff-71bc69f5ab651e6fba551b5db8e47ad90e5bc9d0016c2b0496204cc8badce98a) script added.

[assign_improve_ids.py](https://github.com/PNNL-CompBio/candleDataProcessing/compare/JJ_hcmi_branch?expand=1#diff-37846375a779a3a8e848733ad31b4949350004391808a3e43cc30ef578ff1a90) script added.

### CODE REVIEW

**Comment by jjacobson95** [comment]
PR is ready. I will remove the CI soon though. This CI did not work due to Github memory constraints.

File: hcmi/sample.tsv
Comment (sgosline) [inline hcmi/sample.tsv:1]: this file is not to schema - we dont need all the HCMI data, but we need the columns to match. `improve_sample_id`, `cancer_type`, etc. all the others should be deleted, and the table should be a csv. cancer types should also be standardized. you can add all the additioinal identifiers in the `other_id` field

**Review by sgosline — CHANGES_REQUESTED** [review]
the sample file still isn't right - check the schema and update it before i merge.

File: hcmi/sample.tsv
Comment (jjacobson95) [reply hcmi/sample.tsv:1]: Sorry about this - will do. Is there a list of cancer types that I should map to for renaming/standardizing?

**Comment by jjacobson95** [comment]
Added samples.csv to hcmi. Also added to figshare.

File: hcmi/sample.tsv
Comment (sgosline) [reply hcmi/sample.tsv:1]: This is better. remove the indexing row, and i will merge. the cancer types require manual annotation, i usually use a dictionary. i'll take a look after it's merged.

Comment (jjacobson95) [reply hcmi/sample.tsv:1]: Updated!  Okay sounds good. Seems like Brian may already have an automated method for annotating cancer types across different datasets. But I'd be happy to update mine if there is a standard naming convention available.

**Review by sgosline — APPROVED** [review]

**Pull request merged**